src/components/rangeFilterInput/index.stories.mdx

```diff
@@ -0,0 +1,114 @@
+import { useState } from 'react';
+import { Canvas, Meta, Story } from '@storybook/addon-docs';
+import { action } from '@storybook/addon-actions';
+
+import { Box } from '@chakra-ui/react';
+
+import RangeFilterInput from './index';
+
+<Meta
+  title="Components/Filter/Range Input"
+  component={RangeFilterInput}
+  argTypes={{
+    size: {
+      options: ['md', 'lg'],
+      control: 'select',
+    },
+    isDisabled: {
+      control: 'boolean',
+    },
+  }}
+  parameters={{
+    actions: ['change'],
+  }}
+/>
+;
+
+export const Template = ({ from, to, unit, ...rest }) => {
+  const [currentValue, setCurrentValue] = useState({ from, to });
+  const onChange = (value, name) => {
+    action('change')({
+      ...currentValue,
+      [name]: { ...currentValue[name], value },
+    });
```

Comment:
This seems a bit off:
![Screenshot 2023-02-22 at 15 04 48](https://user-images.githubusercontent.com/144707/220643703-082df9fa-d829-47a4-8aad-d2e09d889953.png)

Shouldn't the `action` be part of `handleChange` definition and take the `event` as the argument?

Reply:
Yes! I was testing something with that action and left it like that :/